force-app/main/default/classes/FlowControlLoopsCollections.cls

```diff
@@ -29,7 +29,7 @@ public with sharing class FlowControlLoopsCollections {
      * @return "Hello World!" if x greater than y, otherwise return null.
      */
     public static String helloWorld(Integer x, Integer y) {
-        return null; // Replace null with the variable you used to store the result
+        return x > y ? 'Hello World!' : null; // Replace null with the variable you used to store the result
```

Comment:
As long as we don't need to do a null check, this is a great spot to use the ternary operator.